roles/auditd/README.rst

```diff
@@ -0,0 +1,73 @@
+Ansible Role for installation and configuration Auditd.
+Auditd is used for collecting Package security messages and forwarding
+them to a central server.
+
+**Role Variables**
+
+.. zuul:rolevar:: auditd_backup
+   :default: false
+
+Sets the backup to false. If true the role will make a backup from the
+rule files. 
+
+.. zuul:rolevar:: auditd_syslog
+   :default: false
+
+This varaible disables write to rsyslog for auditd.
+
+
+**Variable for Configuration**
+
+.. zuul:rolevar:: auditd_config
+
+.. code-block:: yaml
+
+   - parameter: active
+     value: "{{ 'yes' if auditd_syslog|bool else 'no' }}"
+     config: /etc/audisp/plugins.d/syslog.conf
+
+Configuration for Auditd.
```

Comment:
A list of configuration values to set for auditd. Each entry is a hash with three items:

parameter: The name of the parameter to set
value: The value the parameter should be set to
config: The name of the configuration file in which the parameter is set